src/lib/post-jobs.js

```diff
@@ -117,5 +67,108 @@ module.exports = async (dataRoot, date = moment()) => {
 		console.error(await response.text());
 	}
 
-	console.log('Message sent');
+	console.log('Slack message sent');
+};
+
+const sendDiscordMessage = async (discordPostUrl, postings, totalPosts, date) => {
+	const content = [
+		`**Job Posting Update for ${date.format('MMM D, YYYY')}**`,
+		`This week there are **${totalPosts}** total job postings on the [CTS-NL](<https://ctsnl.ca/jobs/>) Job Postings Page.`,
+		`New job posts for week ${date.format('WW')} of ${date.format('YYYY')}\n`,
+	];
+
+	for (const post of postings) {
+		const url = post.indeed ? `https://www.indeed.com/viewjob?jk=${post.indeed}` : post.link;
```

Comment:
This is probably not needed anymore, but it doesn't hurt.

It's for some backwards compatibility :D